src/ui/libs/DatalensChartkit/components/Control/Items/Items.js

```diff
@@ -488,10 +489,7 @@ BaseControlButton.propTypes = {
     label: PropTypes.string,
     theme: PropTypes.string,
     onChange: PropTypes.func.isRequired,
-    className: PropTypes.string,
```

Comment:
Even if you pass `className` in `BaseControlButton` it will be lost [there](https://github.com/datalens-tech/datalens-ui/pull/976/files/09ead15dc784a63c0acae7d1edc94a20f4d9d4c8#diff-fa9b3120811aeee9cca6f9c03bae279959a8c44acb511c19371b7a9588fe5aa9L465) and will not be passed to `Button`

Reply:
We decided to leave the deletion